jwst/resample/resample_spec_step.py

```diff
@@ -61,11 +62,10 @@ def process(self, input):
             kwargs['blendheaders'] = self.blendheaders
 
         kwargs['allowed_memory'] = self.allowed_memory
-        kwargs['weight_type'] = str(self.weight_type)
```

Comment:
Is this going away because we want it to come from the drizpars file, instead of the step spec?

Reply:
I removed that line because now weight_type is set to 'None' in spec and if the user does not set it, then the value in the drizpar reference file is use.  In a fews lines above this: kwargs = self.get_drizpars(ref_filename, input_models)
the kwargs['wht_type'] is filled in.

Reply:
That's what I thought. Just wanted to confirm.